admin/zabbix3/Makefile

```diff
@@ -8,12 +8,21 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=zabbix3
-PKG_VERSION:=3.0.0beta1
+<<<<<<< HEAD
```

Comment:
There are merge conflict leftovers in the last commit. At least from line 10 and line 50 onward...

And you need to sign-off-by your commits.